app/patreon.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express')
+const router = express.Router()
+
+const passport = require('passport')
+const PatreonStrategy = require('passport-patreon').Strategy
+
+passport.use(
+  new PatreonStrategy(
```

Comment:
as I was playing with this, I realized that `passport-patreon` is little more than a thin wrapper over https://github.com/jaredhanson/passport-oauth2 

Although this library is old, it has test coverage and seems to work, so it may be about as mature as it seems abandoned :-p